.env.example

```
@@ -0,0 +1,5 @@
+DB_HOST=localhost   
+DB_PORT=5432
+DB_USER=postgres
+DB_PASS=docker
```

Generally, we don't commit the db_pass, even in the .env.example. The correct way is to use like my suggestion
```suggestion
DB_PASS=
```

---

sure, got it!